Decred Wallet/Features/More/More Features/Settings/SettingsController.swift

```diff
@@ -131,6 +131,14 @@ class SettingsController: UITableViewController  {
         self.tableView.reloadData()
     }
     
+    @objc override func dismissView() {
```

Comment:
we already have this function as an extension, why not use it?

Reply:
I override to disable the animation when the controller dismiss

Reply:
any reason why the animation was removed? because every other more menu uses animation

Reply:
I am wrapping setting controller inside UINavigationController and presenting it. Because animation of presenting is different from animation of pushing other items in the menu so I disabled the animation But I found another issue here so I will get to you after I fix that